src/network/networkd-link.c

```diff
@@ -721,10 +721,15 @@ void link_check_ready(Link *link) {
                     !link->ipv4ll_route)
                         return;
 
-        if (link_ipv6ll_enabled(link))
+        if (link_ipv6_enabled(link)) {
+
                 if (in_addr_is_null(AF_INET6, (const union in_addr_union*) &link->ipv6ll_address) > 0)
                         return;
 
+                if (link_ipv6_accept_ra_enabled(link) && !link->ndisc_configured)
```

Comment:
I meant that nowhere in `link_check_ready` it checks if the device is a bridge, but the commit log makes it sound like the change is bridge specific. Thanks for the pointer -- if `link_ipv6_enabled()` *always* returns false for a bridge this sounds wrong -- how would you ever configure a bridge component which does have RA/DHCP configured on deliberately? This is a problem in the existing code of course, not in this PR. Hence my suggestion to cover both cases with tests, as this is just prone to causing regressions.